tests/test_strategy_one.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+import unittest
+from game.player import Player
+from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
+
+class TestPlayer(unittest.TestCase):
+    def setUp(self):
+        self.strategy_one = StrategyOne()
+
+    def test_strategy_one_on_sixth_column(self):
```

Comment:
Lets revisit the cases you are testing on our Wed talk. Can you add the item to the agenda?